canonical/maas-react-components
# build: create lint rules

## Done
- build: create lint rules

This config reflects the current configuration in MAAS UI

## QA
Run `npm run lint` `npm run lint:fix` `npm run format` and make sure no errors were thrown
